src/main/resources/data-repository-openapi.yaml

```diff
@@ -980,6 +980,24 @@ paths:
           schema:
             type: object
 
+
+  '/api/repository/v1/configuration':
+    get:  ## retrieve
+      description: Retrieve the repository configuration information
+      operationId: retrieveRepositoryConfig
+      tags:
+      - repository
+      responses:
+        200:
+          description: Repository Config
+          schema:
+            $ref: '#/definitions/RepositoryConfigurationModel'
+        403:
```

Comment:
This will be an open endpoint. Actually now that I think about it we shouldn't have it under `/api/repository`.. can you move it to be just at `/configuration` similar to how we are doing the /status endpoint?

Reply:
should it be an unauthenticated endpoint like status us?